lib/modules/manager/gomod/extract.spec.ts

```diff
@@ -40,5 +40,62 @@ describe('modules/manager/gomod/extract', () => {
       expect(res).toHaveLength(58);
       expect(res?.filter((e) => e.skipReason)).toHaveLength(0);
     });
+
+    it('extracts multi-line replace', () => {
```

Comment:
it could be
```
 replace
 (`
 )
```

or 

```
```
replace k8s.io/api => k8s.io/api v0.0.0-20191114100352-16d7abae0d2a

replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20191114105449-027877536833

replace k8s.io/apimachinery => k8s.io/apimachinery v0.16.5-beta.1
```

i dont know what other cases there are , but there might be even a combination (not sure its allowed u can test it)

also the replace doesnt always have a version, sometimes its a path like this: `replace dep.io => ../../somefolder`
make sure u skip those if its not already

Reply:
all this FYI to make sure ur code works , maybe add extra tests to make sure

Reply:
The first is not a valid scenario.
Will add more tests